### PR TITLE
Repair /admin

### PR DESCRIPTION
**Pull request by pahaz**

### CODE REVIEW

**Review by dkoviazin — APPROVED** [review]
Ok

File: apps/condo/domains/common/utils/firebase.front.utils.js [hunk 1]
```diff
@@ -18,8 +19,9 @@ try {
 
 const isFirebaseConfigValid = !!FIREBASE_CONFIG.apiKey
 
-export default firebase
+const auth = firebase.auth
 export {
+    auth,
```
Comment (Dimitreee) [inline apps/condo/domains/common/utils/firebase.front.utils.js:24]:
```suggestion
    auth: firebase.auth,
```

File: apps/condo/domains/common/utils/firebase.front.utils.js [hunk 2]
```diff
@@ -18,8 +19,9 @@ try {
 
 const isFirebaseConfigValid = !!FIREBASE_CONFIG.apiKey
 
-export default firebase
+const auth = firebase.auth
```
Comment (Dimitreee) [inline apps/condo/domains/common/utils/firebase.front.utils.js:22]:
```suggestion
```

File: apps/condo/package.json [hunk 3]
```diff
@@ -19,24 +19,24 @@
...
-    "@keystonejs/fields-color": "^1.0.7",
-    "@keystonejs/fields-content": "^9.0.2",
-    "@keystonejs/fields-location-google": "^3.0.2",
-    "@keystonejs/fields-wysiwyg-tinymce": "^5.3.13",
-    "@keystonejs/file-adapters": "^7.0.3",
-    "@keystonejs/keystone": "^17.1.1",
-    "@keystonejs/list-plugins": "^7.1.4",
-    "@keystonejs/server-side-graphql-client": "^1.1.2",
-    "@keystonejs/utils": "^6.0.1",
+    "@keystonejs/adapter-knex": "^13.2.0",
```
Comment (Dimitreee) [inline apps/condo/package.json:22]: why do we need this updates?

File: apps/condo/pages/_app.tsx [hunk 4]
```diff
@@ -86,12 +81,12 @@ async function messagesImporter (locale) {
     return { ...locale_data.default }
 }
 
+const SSR = true
```
Comment (Dimitreee) [inline apps/condo/pages/_app.tsx:84]: ??

File: apps/condo/pages/auth/register.tsx [hunk 5]
```diff
@@ -14,21 +14,20 @@ import { useIntl } from '@core/next/intl'
 import { useAuth } from '@core/next/auth'
 
 import { TopMenuOnlyLayout } from '@condo/domains/common/components/containers/BaseLayout'
-import firebase, { isFirebaseConfigValid } from '@condo/domains/common/utils/firebase.front.utils'
+import { auth, isFirebaseConfigValid } from '@condo/domains/common/utils/firebase.front.utils'
```
Comment (Dimitreee) [inline apps/condo/pages/auth/register.tsx:17]:
```suggestion
import { auth as firebaseAuth, isFirebaseConfigValid } from '@condo/domains/common/utils/firebase.front.utils'
```

File: apps/condo/pages/auth/register.tsx [hunk 6]
```diff
@@ -14,21 +14,20 @@ import { useIntl } from '@core/next/intl'
...
 
 const AuthContext = createContext({})
 
 function AuthState ({ children, initialUser }) {
     const [user, setUser] = useState(initialUser || null)
 
     useEffect(() => {
-        return firebase.auth().onAuthStateChanged(async function (user) {
-            console.log('onAuthStateChanged', user)
+        return auth().onAuthStateChanged(async function (user) {
```
Comment (Dimitreee) [inline apps/condo/pages/auth/register.tsx:28]:
```suggestion
        return firebaseAuth().onAuthStateChanged(async function (user) {
```

File: packages/@core.next/package.json [hunk 7]
```diff
@@ -1,8 +1,9 @@
 {
   "name": "@core/next",
   "version": "1.0.0",
+  "main": "./index.js",
   "dependencies": {
-    "@apollo/client": "^3.2.5",
+    "@apollo/client": "^3.3.14",
```
Comment (Dimitreee) [inline packages/@core.next/package.json:6]: Why do we need this update?

Comment (pahaz) [reply packages/@core.next/package.json:6]: We should stay up to date with apollo

File: apps/condo/pages/_app.tsx (hunk 4 above)
Comment (pahaz) [reply apps/condo/pages/_app.tsx:84]: To enable/disable SSR in one place %)

**Review by Dimitreee — APPROVED** [review]

File: apps/condo/domains/common/utils/firebase.front.utils.js (hunk 1 above)
Comment (pahaz) [reply apps/condo/domains/common/utils/firebase.front.utils.js:24]: SyntaxError %)

**Pull request merged**